build/openwisp_dashboard/load_init_data.py

```diff
@@ -13,5 +15,44 @@
 
 User = get_user_model()
 
-User.objects.filter(is_superuser=True).exists() or \
-    User.objects.create_superuser("admin", "admin@example.com", "admin")
+
+def create_admin():
+    '''
+    Creates superuser `admin` if it does not exist.
+    '''
+    User.objects.filter(is_superuser=True).exists() or \
+        User.objects.create_superuser("admin", "admin@example.com", "admin")
+
+
+def get_radius_org():
+    '''
+    Fetch default organization.
+    '''
+    return Organization.objects.get(name=os.environ['FREERADIUS_ORGANIZATION'])
+
+
+def set_default_radius_token(radiusOrg):
+    '''
+    Set FREERADIUS_TOKEN to the given organization.
+    '''
+    radiusConf = OrganizationRadiusSettings.objects \
+                                           .filter(organization=radiusOrg)
+    if not radiusConf.exists():
+        radiusConf = OrganizationRadiusSettings()
+        radiusConf.organization = radiusOrg
+        radiusConf.token = os.environ['FREERADIUS_TOKEN']
```

Comment:
why don't we just let the model generate the token?
It feels weird to have an env var for something that is then managed in the DB

Reply:
Removed this, It was used previously, not in use anymore! :smile: